supportedBackends/local/src/main/scala/cromwell/backend/impl/local/LocalJobExecutionActor.scala

```diff
@@ -32,7 +33,7 @@ object LocalJobExecutionActor {
 
 class LocalJobExecutionActor(override val jobDescriptor: BackendJobDescriptor,
                              override val configurationDescriptor: BackendConfigurationDescriptor)
-  extends BackendJobExecutionActor with SharedFileSystem {
+  extends BackendJobExecutionActor with SharedFileSystem with ServiceRegistryClient {
```

Comment:
I started adding runtime attributes metadata for local backend and then rollbacked but this was left behind